lib/weddell/client/subscriber/stream.ex

```diff
@@ -132,10 +132,16 @@ defmodule Weddell.Client.Subscriber.Stream do
   """
   @spec recv(stream :: t) :: Enumerable.t
   def recv(stream) do
-    GRPCStub.recv(stream.grpc_stream)
-    |> Stream.map(fn response ->
-      response.received_messages
-      |> Enum.map(&Message.new/1)
-    end)
+    case GRPCStub.recv(stream.grpc_stream) do
+      {:ok, recv} ->
+        recv
+        |> Stream.map(fn reply ->
+          case reply do
+            {:ok, response} -> Enum.map(response.received_messages, &Message.new/1)
+            {:error, _} -> []
+          end
+        end)
+      {:error, _} -> []
```

Comment:
It seems like we may want to surface these errors somehow instead of just swallowing them. That said the existing code didn't really make an effort to do that either :P. Something to think about, definitely open to suggestions :D.

Reply:
I was bitten pretty quickly by this when I tried to create a consumer that listened on a non-existent subscription 🤦‍♂️. So I went ahead and took a stab at it. Here's what I'm thinking: https://github.com/cjab/weddell/pull/2

This at least very loudly lets you know that your subscription isn't quite right :D.